galleries/views.py

```diff
@@ -1,12 +1,17 @@
 from django.shortcuts import render,get_object_or_404
 from .models import Gallery, Tag
-
+from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
 # Create your views here.
 def galleries(request):
     galleries = Gallery.objects.all()
```

Comment:
I'm not sure how does this work, but you essentially try to fetch all records to memory and then paginate them before displaying to the user.
Maybe django has some sort of a proxy which does not fetch all records until accessed (then that would be fine).
If not then it's still not the best idea to fetch all the objects. It can lead to serious memory issues